sim/mage/blast_wave.go

```diff
@@ -43,59 +43,57 @@ func (mage *Mage) registerBlastWaveSpell() {
 				baseDamage *= sim.Encounter.AOECapMultiplier()
 				spell.CalcAndDealDamage(sim, aoeTarget, baseDamage, spell.OutcomeMagicHitAndCrit)
 			}
-			if targetCount > 1 {
-				mage.FlamestrikeBW.Cast(sim, target)
-			}
+			// if targetCount > 1 {
```

Comment:
Should this commented code be removed?

Reply:
Yes. I moved the Improved Flamestrike logic into the talent and commented the previous code during testing. Will remove it, as it seems to work as I want.